### PR TITLE
Added CRUD to feature file

### PR DESCRIPTION
**Pull request by d4vidmc**

### CODE REVIEW

File: features/project.feature [hunk 1]
```diff
@@ -1,6 +1,91 @@
...
+Feature: Working with Projects on pivotal tracker
 
-  Scenario: Get Project
-    Given I make a connection
-    When I get all projects
-    Then I validate project list is retrieved
\ No newline at end of file
+  Under this feature is covered all CRUD methods available under pivotaltracker.com API documentation.
+
+  Each scenario will have a starting context the Given, a description of an action the When and finally the checked outcome with assertion.
```
Comment (88mary256) [inline features/project.feature]: no more 80 lenght per line

File: features/steps/project_steps.py [hunk 2]
```diff
@@ -1,25 +1,241 @@
...
+    """
+    :type context: behave.runner.Context
+    :type id: str
+    """
+    raise NotImplementedError(u'STEP: And validate the same project\'s <id>')
+
+
+@given("an attempt to delete a project named {name}")
+def step_impl(context, name):
+    """
```
Comment (88mary256) [inline features/steps/project_steps.py:257]: delete this comments in all files

File: config.yml [hunk 3]
```diff
@@ -2,4 +2,6 @@ app:
  #options in level
  token: ""
  base_url: "https://www.pivotaltracker.com/services/v5"
- logger_path:
\ No newline at end of file
+ logger_path: "src/resources/logs/pivotal_api_log.log"
```
Comment (d4vidmc) [inline config.yml]: change path to logs in the root

File: features/project.feature [hunk 4]
```diff
@@ -1,6 +1,91 @@
...
+  Scenario Outline: Update a project
+    Given an attempt to updata a project's <oldname>
+    When User will call the put method
+      And a new project's <newname> is specified
+    Then the project's <oldname> no longer exits
+      And the project's <newname> exists
+      And validate the same project's <id>
+    Examples:
+      | id     | oldname     | newname     |
+      | 123456 | David       | Dav1d       |
```
Comment (d4vidmc) [inline features/project.feature]: do not hardcode the project id

File: features/project.feature [hunk 5]
```diff
@@ -1,6 +1,91 @@
...
+    Given an attempt to delete a project named <name>
+    When User will call the delete method
+      And a the project's <name> is specified to be deleted
+    Then the project's <name> no longer exits
+      And validate the project's <id> is no longer valid
+    Examples:
+      | id     | name        |
+      | 123456 | David       |
+      | 123456 | David MC    |
+      | 123456 | david       |
```
Comment (d4vidmc) [inline features/project.feature]: do not hardcode the project ids

File: src/core/request_handler.py [hunk 6]
```diff
@@ -10,11 +12,13 @@ def __init__(self, header, base_url):
     def get_request(self, url):
         return requests.get(url=url, headers=self.header)
 
-    def post_request(self, url):
-        return requests.post(url=url, headers=self.header)
+    def post_request(self, url, name):
+        body = {"name": name}
```
Comment (d4vidmc) [inline src/core/request_handler.py]: get the body from the parameter

File: src/core/utils/logger.py [hunk 7]
```diff
@@ -1,6 +1,29 @@
...
+
+
+class logger_pivotal:
 
     def __init__(self):
         logging.basicConfig(level=logging.DEBUG)
-        logging.getLogger(__name__)
\ No newline at end of file
+        self.log = logging.getLogger(__name__)
+        self.config = PropertyReader()
```
Comment (d4vidmc) [inline src/core/utils/logger.py]: Apply singleton pattern to this util

File: .gitignore [hunk 8]
```diff
@@ -1,3 +1,4 @@
 /.idea/
+/venv/
 *.pyc
 %allure_result_folder%
```
Comment (d4vidmc) [inline .gitignore]: add logs folder

**Review by d4vidmc — COMMENTED** [review]
Request changes

**Pull request merged**